### ISSUE TITLE
Detected by Windows 10

### ISSUE THREAD

**Issue by pbeneteau** [goliate/hidden-tear#2]
This ransomware is actually detected by Windows 10

**Comment by cozyplanes** [comment]
yup. now many antiviruses are detecting it. better use it only for educational purposes

**Comment by jeanjm21** [comment]
Sou estudante de ti.

E sou iniciante na área de pesquisa de malwares.

Eu gostaria de saber como eu executo esse malware na minha máquina de teste

**Comment by Kruckash** [comment]
@jeanjm21, you could start by learing English or how to use google translate.
If you don't know how to \run\ the malware, then maybe it's not for you? It's pretty well documented imo..

And to you people (@stealdre) who say it detected on Windows 10.. No shit, it's been publicly available for at least 2 years.. Use your..eh, brains.
But thanks for letting us all know 👍

**Comment by giwiro** [comment]
Hi, i found a bypass that consist on signing the executable with fake credentials. [https://github.com/HackerFantastic/Public/blob/master/tools/bypassavp.sh](url)
